sql/2025/webassembly/page_rankings.sql

```diff
@@ -0,0 +1,16 @@
+# Query to get count of pages which use wasm at page ranking intervals.
+
+SELECT
+  client,
+  _rank AS rank,
+  COUNT(DISTINCT page) AS pages
+FROM
+  `httparchive.crawl.requests`,
+  UNNEST([1000, 10000, 100000, 1000000, 10000000]) AS _rank
```

Comment:
Need an extra category in here as we've gone over 10million pages since last last WASM chapter:


```suggestion
  UNNEST([1000, 10000, 100000, 1000000, 10000000, 100000000]) AS _rank
```